src/main/java/mate/controller/car/CreateCarController.java

```diff
@@ -0,0 +1,41 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/create")
```

Comment:
<img width="1061" alt="Screen Shot 2022-09-21 at 8 18 54 PM" src="https://user-images.githubusercontent.com/90692308/191570002-60b562f8-591a-4fcc-bb58-015f5d401a48.png">
Read the checklist more carefully: https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice ! Fix it every where.